pkg/networkmanager/firewall.jsx

```diff
@@ -943,7 +943,7 @@ export class Firewall extends React.Component {
                           <BreadcrumbItem isActive>{_("Firewall")}</BreadcrumbItem>
                       </Breadcrumb>}
                   additionalGroupedContent={
-                      <PageSection id="firewall-heading" className="firewall-heading" variant={PageSectionVariants.light}>
+                      <PageSection id="firewall-heading" variant={PageSectionVariants.light} className="firewall-heading">
```

Comment:
Shouldn't this one also have `ct-pagesection-mobile`? If not, then we can revert this change?

Reply:
No, I think it needs padding as it has breadcrumbs, IIRC.